components/layout/Layout.tsx

```diff
@@ -73,17 +73,25 @@ LayoutProps) {
           <div className="relative mx-auto flex h-full max-w-screen-xl flex-row items-center justify-between px-5 lg:justify-center lg:space-x-10">
             <div className="flex flex-row items-center space-x-3 whitespace-nowrap">
               <div className="flex h-6 flex-row items-center rounded-full bg-white px-2 text-sm font-bold text-primary">
-                {lang === 'en' ? 'NEW' : 'NUEVO'}
+                {lang === 'en' ? 'NEW' : lang === 'fr' ? 'NOUVEAU' : 'NUEVO'}
```

Comment:
please extract a function from here, using multiple ternary operators is a often a bad idea